### DISCUSSION TITLE
docs/0.4/deploy/kubernetes

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#107] [Giscus]
# docs/0.4/deploy/kubernetes

Kubernetes Deployment A Helm chart provided for kubernetes deployment. The chart bootstraps a Gorse deployment on a Kubernetes cluster using the Helm package manager.

https://gorse.io/docs/0.4/deploy/kubernetes.html

**Comment by arunas-bendikas** [comment]
I would recommend not to hardcode config files in gorse-master pod in the helm chart. Allow user to overwrite config or pod env variables in their own values.yaml file.
